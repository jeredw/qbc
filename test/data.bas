data 1,,2
data foo, some words here, "this is a thing", 42.6, well! interesting,,
data how ' exciting is this : print "very"
