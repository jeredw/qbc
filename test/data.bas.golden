(program 
  (statement 
    (data_statement data 
      (data_item 1) , data_item , 
      (data_item 2))) \n 
  (statement 
    (data_statement data 
      (data_item foo) , 
      (data_item some words here) , 
      (data_item "this is a thing") , 
      (data_item 42.6) , 
      (data_item well! interesting) , data_item , data_item)) \n 
  (statement 
    (data_statement data 
      (data_item how ' exciting is this))) : 
  (statement 
    (print_statement print 
      (expr 
        (literal "very")))) \n 
  (if_block_statement if 
    (expr 
      (literal 1)) then \n 
    (then_block 
      (statement 
        (print_statement print 
          (expr 
            (literal "hi")))) : 
      (statement 
        (data_statement data 
          (data_item 1) , 
          (data_item 2) , 
          (data_item 3))) : 
      (statement 
        (print_statement print 
          (expr 
            (literal "bye")))) \n) 
    (end_if_statement end if)) \n)
